File: wxrtc/src/main/java/com/wx/rtc/bean/CallCommand.java
package com.wx.rtc.bean;

public @interface CallCommand {
    String INVITE = "invite";
    String CANCEL = "cancel";
    String ACCEPT = "accept";
    String REJECT = "reject";
    String LINE_BUSY = "line_busy";
    String HANG_UP = "hang_up";
    String NEW_INVITATION_RECEIVED = "new_invitation_received";
    String INVITATION_CANCELED = "invitation_canceled";
    String INVITATION_ACCEPTED = "invitation_accepted";
    String INVITATION_REJECTED = "invitation_rejected";
    String INVITATION_NO_RESP = "invitation_no_resp";
    String INVITATION_LINE_BUSY = "invitation_line_busy";
    String CALL_END = "call_end";
}

File: wxrtc/src/main/java/com/wx/rtc/bean/CallCommand.kt
package com.wx.rtc.bean

internal annotation class CallCommand {
    companion object {
        const val INVITE: String = "invite"
        const val CANCEL: String = "cancel"
        const val ACCEPT: String = "accept"
        const val REJECT: String = "reject"
        const val LINE_BUSY: String = "line_busy"
        const val HANG_UP: String = "hang_up"
        const val NEW_INVITATION_RECEIVED: String = "new_invitation_received"
        const val INVITATION_CANCELED: String = "invitation_canceled"
        const val INVITATION_ACCEPTED: String = "invitation_accepted"
        const val INVITATION_REJECTED: String = "invitation_rejected"
        const val INVITATION_NO_RESP: String = "invitation_no_resp"
        const val INVITATION_LINE_BUSY: String = "invitation_line_busy"
        const val CALL_END: String = "call_end"
    }
}
